setup.py

```diff
@@ -39,37 +41,40 @@ def setup_package():
         version=VERSION,
         description=DESCRIPTION,
         long_description=LONG_DESCRIPTION,
-        long_description_content_type='text/markdown',
+        long_description_content_type="text/markdown",
         author=AUTHOR,
         author_email=AUTHOR_EMAIL,
         maintainer=MAINTAINER,
         maintainer_email=MAINTAINER_EMAIL,
         license=LICENSE,
         url=URL,
-        classifiers=['Development Status :: 3 - Alpha',
-                     'Intended Audience :: Developers',
-                     'License :: OSI Approved :: Apache Software License',
-                     'Topic :: Database',
-                     'Programming Language :: Python',
-                     'Programming Language :: Python :: 3',
-                     'Programming Language :: Python :: 3.5',
-                     'Programming Language :: Python :: 3.6',
-                     'Programming Language :: Python :: 3.7',
-                     ],
-        packages=find_packages(exclude=['*tests*']),
+        classifiers=[
+            "Development Status :: 3 - Alpha",
+            "Intended Audience :: Developers",
+            "License :: OSI Approved :: Apache Software License",
+            "Topic :: Database",
+            "Programming Language :: Python",
+            "Programming Language :: Python :: 3",
+            "Programming Language :: Python :: 3.5",
+            "Programming Language :: Python :: 3.6",
+            "Programming Language :: Python :: 3.7",
+        ],
+        packages=find_packages(exclude=["*tests*"]),
         install_requires=[
-            'urllib3<1.25,>=1.21.1',
-            'presto-python-client>=0.6.0',
-            'pandas>=0.22.0',
-            'td-client>=0.12.0',
-            'pytz>=2018.5',
+            "urllib3<1.25,>=1.21.1",
+            "presto-python-client>=0.6.0",
+            "pandas>=0.22.0",
+            "td-client>=0.12.0",
+            "pytz>=2018.5",
         ],
         extras_require={
-            'spark': ['pyspark>=2.4.0', 'pyarrow>=0.11.0'],
-        })
+            "spark": ["pyspark>=2.4.0", "pyarrow>=0.11.0"],
+            "dev": ["black", "isort", "flake8"],
+        },
```

Comment:
We can create `test` dependency with `["pytest"]` and modify `noxfile.py` as:

```py
@nox.session
def tests(session):
    session.install(".[dev,test,spark]")
    session.run("pytest", "-v")
```

Reply:
fixed 4f3f37c